packages/core/src/components/Badge/badge.example.html

```diff
@@ -1,3 +1,35 @@
-<span class="ds-c-badge {{modifier}}">5</span>
-<span class="ds-c-badge {{modifier}}">Badge label</span>
-<span class="ds-c-badge {{modifier}} ds-u-font-size--base">Badge label</span>
+<div class="example--wrapper">
```

Comment:
Wondering if we should add labels above the badges like we're doing for buttons?

Reply:
The only reason why I didnt is because the Badges already have the name of the color/type of Badge (i.e. Primary, Warning etc). You cant do this with buttons because the labels refer to hover, focus, active etc